.codecov.yml

```diff
@@ -19,13 +19,13 @@ coverage:
     project:
       default:
         # commits below this threshold will be marked as failed
-        target: '58%'
+        target: '85%'
         # how much we allow the coverage to drop
-        threshold: '5%'
+        threshold: '2%'
```

Comment:
Also reduced this to 2%, but didn't change the one in patch as I am not sure how that works for that case.